src/connect.rs

```diff
@@ -85,7 +53,7 @@ mod encryption {
 pub use self::encryption::MaybeTlsStream;
 
 #[cfg(not(feature = "tls"))]
-mod encryption {
+pub(crate) mod encryption {
     use futures::{future, Future};
```

Comment:
This imports are not used

Reply:
Fixed now. Forgot to try building without default features.